jwst/coron/imageregistration.py

```diff
@@ -216,17 +216,20 @@ def align_models(reference, target, mask):
     output_model = QuadModel(quad_shape)
     output_model.update(target)
 
+    # Compute the shifts of the PSF ("target") images relative to
+    # the science ("reference") image in the first integration
+    _, shifts = align_array(
```

Comment:
This actually shifts the data, in addition to computing the shifts, which is unnecessary since it will be shifted later.  It looks like this is the only place align_array is called, so we should either revise it to separate out shift calculation and the actual data shifting, or else remove the function and just call the necessary steps here.

Reply:
I agree that align_array() can be removed and `align_fourierLSQ` can be called directly.

Reply:
I removed the part where the shifting of the data is happening, nonetheless the conditionals are needed so I modified the function